Glassdoor data cleaning & Indeed data scraping

lijning opened the pull request:
Glassdoor data cleaning initialized & Indeed data scraping bug fixed.

XinyuanHe-columbia committed:
glass door data cleaning

babbbble reviewed (approved):
i think it is perfect

lijning committed:
Fixing data scaping bug

1. Fixing bug occuring when searching with job titles: white space
handled.
2. "Data engineer" dataset with title speficifed pulled.
